2025/build-tooling/demo/final/eslint.config.js

```diff
@@ -7,7 +7,7 @@ import reactRefresh from "eslint-plugin-react-refresh";
 
 export default [
   eslint.configs.recommended,
-  ...tsEslint.configs.recommendedTypeChecked,
+  ...tsEslint.configs.recommended,
```

Comment:
`recommendedTypeChecked` made things too verbose for this demo

Reply:
Sounds good – the one of the girls of this presentation was to show how Esri builds apps – so I tried to keep config balance between simple and matching what JavaScript API does